master.js

```diff
@@ -353,8 +353,13 @@ class AlAzureMaster {
                     master._azureContext.log.warn('Health check failed with: ',  errStatus);
                     status = master.errorStatusFmt('ALAZU000004', errStatus);
                 } else {
-                    master._azureContext.log.warn('Health check failed with',  errStatus.details);
-                    status = errStatus;
+                    if(errStatus.details){
+                        master._azureContext.log.warn('Health check failed with',  errStatus.details);
+                        status = errStatus;
+                    }else{
+                        status = master.errorStatusFmt('ALAZU000005', errStatus);
+                        status = errStatus;
```

Comment:
is this redundant (`status` being assigned twice)?

Reply:
good catch scott, made changes and pushed again